bigtable/google/cloud/bigtable/table.py

```diff
@@ -29,6 +29,18 @@
 from google.cloud.bigtable.row_data import PartialRowsData
 
 
+# Maximum number of mutations in bulk
+_MAX_BULK_MUTATIONS = 100000
```

Comment:
@tseaver It is documented back-end limit: https://cloud.google.com/bigtable/docs/reference/data/rpc/google.bigtable.v2#mutaterowsrequest